### ISSUE TITLE
Not logging in using QR Code

### ISSUE THREAD

**Issue by OwaisKhatri** [nashaofu/dingtalk#350]
Hi,
I am not able to login using QR Code Scanning to my laptop using Chrome Extension, Desktop App and Web App(https://im.dingtalk.com/).
I am logged in on my Android App.
It is also not showing any error.
Thanks.

**Comment by nashaofu** [comment]
I didn't reproduce it. I can use it normally

**Comment by raffifu** [comment]
Have same issue cannot login using qr code

**Comment by OwaisKhatri** [comment]
@nashaofu

**Comment by nashaofu** [comment]
Please use your browser to visit https://im.dingtalk.com/, and then check whether you can log in, if you can not log in, please contact Alibaba

**Comment by raffifu** [comment]
Yeah, i think you're right. There is no problem in your app, because your app use https://im.dingtalk.com/ and i am not able to log in even use my browser

**Comment by jerng** [comment]
Same - I think Chrome extension, Linux .deb, and im.dingtalk.com are the same - not sure what's up. Overseas user.

**Comment by tehsunnliu** [comment]
Hi, I'm facing the same problem. After scanning the QR Code from my Android device DingTalk nothing happens on DingTalk Web nor on DingTalk Desktop. I've tried on Ubuntu, Windows, and Browser. No error message either. When I try to login with password it says its depricated. 
I think password login should be enabled for at least overseas users. I'm currently located in India using DingTalk on my mobile device.
I've also tried VPN to login to DingTalk Web/Desktop version but no success. Hope this gets resolved soon. 
Its being like this since past few months.

**Comment by code-fury** [comment]
Is there any fix for this?
